libraries/ui/src/Card.tsx

```diff
@@ -29,13 +28,15 @@ export const Card: React.FC<CardProps> = ({
   isEntireCardClickable = false,
   isExternalUrl = false,
   className = '',
-  imageClassName = '',
+  imageSrc,
   subtitleClassName = '',
+  imageClassName = '',
+  imageNode,
 }) => {
   const Wrapper = isEntireCardClickable ? 'a' : 'div';
   const wrapperClassName = clsx(
-    'card flex flex-col items-start min-w-min-width transition-transform duration-200',
-    isEntireCardClickable && 'hover:scale-[1.01] hover:container-elevated',
```

Comment:
Moved the box-shadow to the CourseCard component, because it doesn't look good on the version with no border